docs/sql/endpoints.md

```diff
@@ -113,3 +113,67 @@ Explain:
   }
 }
 ```
+
+## Cursor
+
+### Description
+
+To get back a paginated response, use the `fetch_size` parameter. The value of `fetch_size` should be greater than 0, the default value is 1000. A value of 0 will fallback to a non-paginated response.
```

Comment:
greater than 0. The default value is 1,000.

Reply:
Done.